lib/src/model/keychain.dart

```diff
@@ -209,6 +210,42 @@ class Keychain with _$Keychain {
       'verificationMethod': verificationMethods,
     };
   }
+
+  ({Uint8List secret, List<AuthorizedKey> authorizedPublicKeys})
```

Comment:
The second parameter here should not be a list of AuthorizedKey but a list a PulicKey. The Authorized key is the output of the function

Reply:
`publicKeys` is a param of the method
`({Uint8List secret, List<AuthorizedKey> authorizedPublicKeys})`is a output tupple with secret and  authorizedPublicKeys

i think it's ok in my side

Reply:
Right, my bad I thought ot was input parameters